charts/topolvm/templates/lvmd/role.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.lvmd.managed }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: psp:lvmd
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "topolvm.labels" . | nindent 4 }}
+rules:
+  - apiGroups: ["policy"]
+    resources: ["podsecuritypolicies"]
+    verbs: ["use"]
+    resourceNames: ["{{ template "topolvm.fullname" . }}-lvmd"]
```

Comment:
Would you concider podSecurityPolicy.create=false?

Reply:
fixed: https://github.com/topolvm/topolvm/pull/302/commits/f9b9be464436414413371578d9fdd48573298fe1